src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -158,6 +158,12 @@ var DOMPropertyOperations = {
         } else if (propertyInfo.hasBooleanValue ||
                    (propertyInfo.hasOverloadedBooleanValue && value === true)) {
           node.setAttribute(attributeName, '');
+        } else if (attributeName === 'value' && node.hasAttribute('value')) {
+          // Use loose coercion to prevent replacement on comparisons like
+          // '3e1' == 30 in Chrome (~52).
+          if (node.value != value) { // eslint-disable-line
+            node.setAttribute(attributeName, '' + value);
```

Comment:
Looks like this is taken care of, because `node.value` on progress is never assigned, so it's out of sync with `props.value`, and it reassigns the attribute.

![progress](https://cloud.githubusercontent.com/assets/590904/18248537/cc559d62-7346-11e6-9699-6d7b8b9e1913.gif)